src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -3454,8 +3454,8 @@ private void recordRelocation(void* location, void* target, ushort fRelocType, u
             Debug.Assert(locationBlock != BlockType.Unknown, "BlockType.Unknown not expected");
 
             // TODO: Arbitrary relocs
-            if (locationBlock != BlockType.Code)
-                throw new NotImplementedException("Arbitrary relocs");
+            // if (locationBlock != BlockType.Code)
```

Comment:
@jkotas This is a little bit confusing because there is the code below processing other cases than BlockType.Code.